src/examples/read_write_fixedlen_string.cpp

```diff
@@ -34,10 +34,10 @@ int main(void) {
         file.createDataSet("ds2", strings_fixed);
 
         // Now test the new interface type
-        //FixedLenStringArray<10> arr(std::vector<std::string>{"0000000", "1111111"});
-        FixedLenStringArray<10> arr{"0000000", "1111111"};
+        FixedLenStringArray<10> arr{"0000000", "1111111"};  // also accepts std::vector
```

Comment:
I would at least ensure that the code compiles when passing a vector. If it is done elsewhere, please ignore this comment.

Reply:
Yep, the unit tests have that ctor :)